packages/suite/src/views/dashboard/components/T2B1PromoBanner.tsx

```diff
@@ -88,7 +90,7 @@ const NextGenerationText = styled.p`
 `;
 
 const NextGenerationTextBlock = styled.span`
-    color: ${({ theme }) => theme.TYPE_WHITE};
+    color: ${THEME.light.TYPE_WHITE};
```

Comment:
Since we are trying to get rid of the old theme slowly, please use the new `textOnPrimary` ✌️